sru-scripts/manual/oracle-sru

```diff
@@ -0,0 +1,80 @@
+#!/bin/bash
+# Manual Oracle upgrade and clean install validation
+set -ex 
+
+SRU_VARS=./sru-scripts/sru-vars.template
+
+if [ ! -f $SRU_VARS ]; then
+   echo "Missing config environment file $SRU_VARS."
+   exit 1
+fi
+
+. $SRU_VARS
+
+parse_args $0 $@
+assert_expected_vars PROPOSED_SCRIPT SAMPLE_CLOUDCONFIG LP_USER SSHOPTS \
+    NETCFG_FILE
+create_samplecloudconfig
+create_setup_proposed_script
+
+
+echo "### BEGIN $UBUNTU_RELEASE"
+
+# TODO: Use the Oracle CLI
+echo "Please manually launch a $release instance in Oracle and enter its IP address"
```

Comment:
I think `$release` here should be replaced by `$UBUNTU_RELEASE`

Reply:
+1, fixed throughout. And also s/setup_proposed.sh/${PROPOSED_SCRIPT}/